modules/react/src/components/export-video/utils.js

```diff
@@ -109,10 +110,9 @@ export function estimateFileSize(frameRate, resolution, durationMs, mediaType) {
   // TODO Read resource from Imgur dev https://stackoverflow.com/questions/23920098/how-to-estimate-gif-file-size
   if (mediaType === 'gif') {
     // ParseInt to turn it from float to int
-    const seconds = parseInt(durationMs / 1000, 10);
-    return `${parseInt(
-      ((resolution[0] * resolution[1] * 6) / (8 * 1024 * 1024)) * (frameRate * seconds) * 0.8,
-      10
+    const seconds = Math.floor(durationMs / 1000);
+    return `${Math.floor(
+      ((resolution[0] * resolution[1] * 6) / (8 * 1024 * 1024)) * (frameRate * seconds) * 0.8
```

Comment:
Can we split this math up into clearer parts? I'm not sure what the numbers here represent. Also, would like the comments in this function to be cleaned up.

Reply:
I'm not sure what you mean. There's an explanation of the formula below the function.